js/main.js

```diff
@@ -0,0 +1,41 @@
+let number;
+let block;
+for(let i=1;i<=15;i++){
+    block=document.createElement('li');
+    block.innerHTML=(`Character ${i}`);
+    block.className=(`${i}`);
+    document.querySelector('.character-list').appendChild(block);
```

Comment:
First, create and HTML and append it once. Because this manipulation DOM is so valuable procedure